### DISCUSSION TITLE
Unable to upload images

### DISCUSSION THREAD

**Discussion by MaryMuse** [mastodon/mastodon#20183] [Q&A]
(second update) - I really need someone to help me t/s this. I've verified my configuration against the example file. I even spun up a new DO droplet now that they updated to 3.5.3 and I'm getting the same thing.

Steps:
1) Go to profile, upload a new 353KB avatar image, click SAVE.
2) Mastodon goes to a "there's been an error" page.
3) I see the image has been uploaded to S3
4) refresh the page/move to a different page, image isn't showing and it's updated.

On the 3.5.3 I don't get the black boxes, but it appears to be a similar issue. Thank you.

(updated -- I found the error logs but am seeing nothing about this.)

I've updated Amazon S3 and I can see files are being sent and written there. However, on Mastodon they show up blank. (See attached)
![Screenshot 2022-11-08 191427](https://user-images.githubusercontent.com/36464631/200712127-d0a121e4-1696-48ec-aed2-df872cff9e98.jpg)

I'm sure there's a setting I'm missing. Can anyone help?

**Comment by abdessalaam** [comment]
Try changing permissions for your mastodon/public/system

sudo chown -R www-data:www-data /home/mastodon/public
